language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/hover/HoverUtil.java

```diff
@@ -349,6 +351,31 @@ private static Hover getFunctionHoverMarkupContent(FunctionSymbol symbol, HoverC
         return hover;
     }
 
+    private static Hover getVariableHoverMarkupContent(VariableSymbol symbol) {
+        Optional<Documentation> documentation = symbol.documentation();
+        if (documentation.isEmpty()) {
```

Comment:
Even If the documentation is empty, we can provide the hover, since that only affects to the description. Hence we can combine this check with line number 361.